alacritty_terminal/src/term/mod.rs

```diff
@@ -933,6 +933,11 @@ impl Term {
                 self.colors[i] = self.original_colors[i];
             }
         }
+        if self.original_font_size != config.font.size && self.original_font_size == self.font_size
```

Comment:
It's probably not worth it to check for `self.original_font_size != config.font.size`, since all that saves us are two assignments which should basically be free. The font itself is only ever changed if it actually changed something anyways.

I'd also like to put this at the end of the method with one newline separating it. Just to separate the different blocks a bit from each other and not have one massive function without a single newline.